packages/web/src/pages/sign-up-page/components/AccountHeader.tsx

```diff
@@ -1,80 +1,173 @@
+import { ImageField } from './ImageField'
+
+type AccountHeaderProps = {
+  mode: 'editing' | 'viewing'
+}
+
+const CoverPhotoBox = ({
+  imageUrl,
+  isEditing
+}: {
+  imageUrl: string | undefined
+  isEditing?: boolean
+}) => (
+  <Box
+    h='100%'
+    w='100%'
+    border='default'
+    css={{
+      backgroundColor: 'lightgray',
+      overflow: 'hidden',
+      ...(imageUrl
+        ? {
+            backgroundImage: `url(${imageUrl})`,
+            backgroundPosition: 'center',
+            backgroundSize: '100%',
+            backgroundRepeat: 'no-repeat, no-repeat'
+          }
+        : {
+            background:
+              'linear-gradient(0deg, rgba(0, 0, 0, 0.20) 0%, rgba(0, 0, 0, 0.00) 100%), #C2C0CC'
+          })
+    }}
+  >
+    {isEditing && !imageUrl ? (
+      <IconImage
+        css={{ position: 'absolute', right: '16px', top: '16px' }}
+        color='staticWhite'
+      />
+    ) : null}
+  </Box>
+)
+
+const ProfileImageAvatar = ({
+  imageUrl,
+  isEditing
+}: {
+  imageUrl: string
+  isEditing?: boolean
+}) => {
+  const { isMobile } = useMedia()
+  const useSmallSize = isEditing || isMobile
+
+  const avatarSize = useSmallSize ? 72 : 120
+  return (
+    <Avatar
+      variant='strong'
+      src={imageUrl}
+      css={{
+        height: avatarSize,
+        width: avatarSize,
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'center',
+        ...(useSmallSize ? { transform: 'translateY(20px)' } : null)
+      }}
+    >
+      {isEditing && !imageUrl ? (
+        <IconCamera size='l' color='staticWhite' />
+      ) : null}
+    </Avatar>
+  )
+}
+
+export const AccountHeader = ({ mode }: AccountHeaderProps) => {
   const coverPhoto = useSelector(getCoverPhotoField)
   const profileImage = useSelector(getProfileImageField)
   const { value: displayName } = useSelector(getNameField)
-  const { value: handle } = useSelector(getHandleField)
+  // const { value: handle } = useSelector(getHandleField)
 
-  const { isMobile } = useMedia()
+  const handle = 'f&$kTheFireNation'
+  const isEditing = mode === 'editing'
 
-  const avatarSize = isMobile ? 72 : 120
+  const { isMobile } = useMedia()
+  const useSmallSize = isEditing || isMobile
 
   return (
-    <Box>
-      <Box h={isMobile ? 96 : 168} css={{ overflow: 'hidden' }}>
-        <Box
-          h='100%'
-          w='100%'
-          border='default'
-          css={{
-            backgroundImage: `linear-gradient(0deg, rgba(0, 0, 0, 0.20) 0%, rgba(0, 0, 0, 0.00) 100%), url(${coverPhoto.url}), url(${coverPhoto.url})`,
-            backgroundPosition: '2.033px -355.342px, center, center',
-            backgroundSize: '100% 571.429%, cover',
-            backgroundRepeat: 'no-repeat, no-repeat',
-            backgroundColor: 'lightgray',
-            filter: 'blur(25px)',
-            overflow: 'hidden'
-          }}
-        />
+    <Box w='100%'>
+      <Box h={useSmallSize ? 96 : 168} css={{ overflow: 'hidden' }} w='100%'>
```

Comment:
Feels like we're doing a lot of manual height and widths. Not necessarily opposed, but it may be a sign that our standardized harmony sizing isn't serving us properly, or that the designs aren't adhering to convention.

Reply:
Maybe this is a hot take on my part but I don't think using preset sizing gains us much tbh. IMO, dimensions are too hard to standardize and trying to fit into a scale probably. 
But on the other hand I do agree that I don't love how we end up with "magic" numbers like this where it's hard to tell if this was just a "this looks good" number or a number derived from something

Reply:
Yeah I think its a fair critique and looking back over this PR I think the biggest smells to me are
- Avatar Sizing. The sizes we're using here feel fairly standard? I was a bit surprised when I went to the avatar sizes and neither of them were even close to the size used here
- Centered container width. While I think dimensions of things lower are probably trickier to nail down, I do think we should consider some standard "top-most layer" container sizes (or maxWidths specifically). I.e. this is the "desktop one centered column container"
I think the rest are probably fine as-is

Reply:
Perfect. That makes sense to me :)

Was also feeling weird about Avatar sizes after running into the same problem. If we're going to have predefined sizes, we should be using those almost exclusively. If not, then it should just accept an arbitrary size